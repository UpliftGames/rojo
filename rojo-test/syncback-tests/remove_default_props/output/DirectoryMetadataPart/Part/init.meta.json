{
  "properties": {
    "CanCollide": false,
    "Massless": false,
    "Anchored": true
  },
  "className": "Part"
}
