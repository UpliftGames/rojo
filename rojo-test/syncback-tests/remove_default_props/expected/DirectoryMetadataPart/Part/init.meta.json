{
  "properties": {
    "Anchored": true
  },
  "className": "Part"
}
